### DISCUSSION TITLE
First time using Kobold and similar softare in general

### DISCUSSION THREAD

**Discussion by aJIekc3D** [LostRuins/koboldcpp#774] [Q&A]
I`m using:
Xeon E5-2690v2
RX 570 8gb
16 RAM
And latest version of koboldcpp (no cuda version) with "Wizard-Vicuna-13B-Uncensored.ggmlv3.q4_K_S" and Vulkan NoAVX2 + 10 GPU layers (can`t understand if it`s even useful cuz 0% GPU and VRAM usage) with average speed 2.9 - 3.1 T/s. Can I impove speed somehow,or that the best i can get with my hardware?

**Comment by LostRuins** [comment]
This model is too old to use with Vulkan. Use a different backend, or get a more up-to-date model.

**Reply by aJIekc3D** [reply]
Which one for example?

**Reply by LostRuins** [reply]
Please try this one: https://huggingface.co/TheBloke/MistRP-Airoboros-7B-GGUF/resolve/main/mistrp-airoboros-7b.Q4_K_S.gguf or this one https://huggingface.co/KoboldAI/LLaMA2-13B-Tiefighter-GGUF/resolve/main/LLaMA2-13B-Tiefighter.Q4_K_S.gguf

**Reply by aJIekc3D** [reply]
It works now,but can`t say if it`s faster at all (around 2.7 T/s) plus i get random symbols instead of 
answer 
![изображение_2024-04-11_162611409](https://github.com/LostRuins/koboldcpp/assets/134833309/361c4d5d-864e-46cd-aa7f-1367347e6ad2)

**Reply by LostRuins** [reply]
which model did you try?

**Reply by aJIekc3D** [reply]
Both, The result is the same

**Reply by LostRuins** [reply]
Alright, maybe try the CLBlast NoAVX2 instead then

**Reply by aJIekc3D** [reply]
Now works as it should, yet still with lower speed than wizard model, but character initialization faster (1200 tokens in 40s with working vulkan vs 250s without it). Also played with gpu layers value (5-20), nothing changed except 2gb RAM usage difference

**Reply by LostRuins** [reply]
Unfortunately if you can't use AVX2 your options are rather limited. What are your PC specs? CPU?

**Reply by aJIekc3D** [reply]
Understood... I wrote pc specs in question description, xeon E5 2690v2
